charts/kargo/templates/api/configmap.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     {{- include "kargo.api.labels" . | nindent 4 }}
 data:
   KARGO_NAMESPACE: {{ .Release.Namespace }}
-  LOG_LEVEL: {{ .Values.api.logLevel }}
+  LOG_LEVEL: {{ quote .Values.api.logLevel }}
```

Comment:
`values.yaml` sets this to `INFO`. If a user set it to `""`, that should be ok because the Go code provides its own default.

Quotes added to avoid https://github.com/argoproj/argo-cd/issues/15566

Same change applied in other configmaps as well.